### ISSUE TITLE
Unwanted text when using sphinx-quickstart

### ISSUE THREAD

**Issue by akshaybabloo** [sphinx-doc/sphinx#3406]
### Problem
I am trying to create a new documentation but when I do that I get some unwanted numbers and question marks

```
Welcome to the Sphinx 1.5.1 quickstart utility.

Please enter values for the following settings (just press Enter to
accept a default value, if one is given in brackets).

Enter the root path for documentation.
[35m> Root path for the documentation [.]: [39;49;00m
```

#### Procedure to reproduce the problem
```
sphinx-quickstart
```

### Environment info
- OS: Windows 10
- Python version: 3.5
- Sphinx version: 1.5.1

**Comment by tk0miya** [comment]
Could you try the checker pasted at #1820?

**Comment by akshaybabloo** [comment]
This is the output:

```
C:\Users\aksha\Anaconda3\python.exe "C:/Users/aksha/Box Sync/MyDrive/Projects/Akshay-GitHub/Car-ND/test.py"
3.5.2 |Anaconda custom (64-bit)| (default, Jul  5 2016, 11:41:13) [MSC v.1900 64 bit (AMD64)]
Windows-10-10.0.14393-SP0
no libedit and readline
colorama enabled
isatty() False
COLORTERM in ENV False
term in ('xterm', 'linux') False
'color' in term False
COLOR TERMINAL: On
----- coloring test -----
Welcome to the Sphinx coloring test utility 1.5-util-2.

Please report a capture of this screen if console is broken.

Normal color text
Red color text
Green color text
Normal color text
Please enter your favorite color (blank to exit): red
red
Please enter your favorite color (blank to exit): green
green
Please enter your favorite color (blank to exit): 
```

But `colorama` is installed on my system and above text were in green color.

**Comment by tk0miya** [comment]
@shimizukawa Could you check this please?

**Comment by shimizukawa** [comment]
OK, I'll check at this night (just now I don't have Win10 env)

**Comment by shimizukawa** [comment]
I got it.
@akshaybabloo please try `pip install pyreadline` to avoid the problem.

Actually, this problem is fixed by #3364 and next version (1.5.3 and 1.6) will be shipped with the fix.

**Comment by akshaybabloo** [comment]
Yes! That worked thanks.
